doc/recipe/preprocessor.rst

```diff
@@ -1512,14 +1509,14 @@ In the most general case, we can set upper limits on the maximum memory the
 analysis will require:
 
 
-``Ms = (R + N) x F_eff - F_eff`` - when no multimodel analysis is performed;
+``Ms = (R + N) x F_eff - F_eff`` - when no multi-model analysis is performed;
 
-``Mm = (2R + N) x F_eff - 2F_eff`` - when multimodel analysis is performed;
+``Mm = (2R + N) x F_eff - 2F_eff`` - when multi-model analysis is performed;
```

Comment:
have you tested these or are they theoretical assumptions? why is there now a factor 2 for the computational efficiency?

Reply:
I don't know where this comes from. Just added the hyphen

Reply:
(I tested and gathered data and got these down empirically) - also, my bad, I saw double, and didn't notice the only change was the dash :eyeglasses: